alpine/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN apk update --no-cache && \
         tini \
         ttf-dejavu \
         openjdk${JAVA_VERSION} \
+        nodejs \
```

Comment:
It only makes sense to install it for OH 5 or newer and not OH 4 (or older) which use the same files.

You can prevent that by adding some conditional logic based on the version.
See the Debian Dockerfile for how this is done ([ref1](https://github.com/openhab/openhab-docker/blob/51ee99a1bf8cd162d57c9decc755bec708427b33/debian/Dockerfile#L85-L87), [ref2](https://github.com/openhab/openhab-docker/blob/51ee99a1bf8cd162d57c9decc755bec708427b33/debian/Dockerfile#L64)).

Is there already a PR for the Matter add-on? Merging this PR depends on if that gets merged.

Reply:
No, there is no Matter PR yet, but I am sure it will get merged given the interest of openHAB maintainers on it.
Not having NodeJS in thr Alpine image available will prevent users from installing and testing the Matter binding from thr marketplace through.